protocol/messenger_backup_test.go

```diff
@@ -737,36 +738,39 @@ func (s *MessengerBackupSuite) TestBackupKeypairs() {
 	s.Require().NoError(err)
 	s.Require().True(accounts.SameKeypairsWithDifferentSyncedFrom(seedKp, dbSeedKp2, false, accounts.SyncedFromBackup, accounts.AccountNonOperable))
 
+	keypairs, err := bob2.settings.GetAllKeypairs()
+	s.Require().NoError(err)
+
 	// Check whether accounts added event is sent
-	expectedAddresses := [][]common.Address{}
-	profileKpWalletAddresses := []common.Address{}
-	seedKpAddresses := []common.Address{}
+	expectedAddresses := []common.Address{}
```

Comment:
Maybe just define `expectedAddresses` and `receivedEventAddresses` as maps?
This will prevent the sorting issues.
```suggestion
	expectedAddresses := make(map[common.Address]struct{}, 0)
```

Reply:
A map is a good idea. I guess a map with a boolean value would be simple like adding a key (address) with a value false and updating it to true on receiving the event. We can remove the length check and another var to hold the received addresses. Ultimately, we can check if all values in the maps are true.

I will update it 👍 Thanks!